src/sonic-config-engine/sonic-cfggen

```diff
@@ -174,13 +182,16 @@ def main():
     parser.add_argument("-d", "--from-db", help="read config from configdb", action='store_true')
     parser.add_argument("-H", "--platform-info", help="read platform and hardware info", action='store_true')
     parser.add_argument("-s", "--redis-unix-sock-file", help="unix sock file for redis connection")
+    parser.add_argument("-i", "--interface", help="print specific port details")
     group = parser.add_mutually_exclusive_group()
     group.add_argument("-t", "--template", help="render the data with the template file")
     group.add_argument("-v", "--var", help="print the value of a variable, support jinja2 expression")
     group.add_argument("--var-json", help="print the value of a variable, in json format")
     group.add_argument("-w", "--write-to-db", help="write config into configdb", action='store_true')
     group.add_argument("--print-data", help="print all data", action='store_true')
     group.add_argument("--preset", help="generate sample configuration from a preset template", choices=get_available_config())
+    group.add_argument("--print-acl", help="print acls", action="store_true")
+    group.add_argument("--print-port", help="print ports details", action="store_true")
```

Comment:
can you use --var-json option? why do we need to introduce new optons?

Reply:
Makes sense. Using --var-json to take care of this. Will update https://github.com/Azure/sonic-utilities/pull/557 to reflect the changes for show command.

Reply:
There is just a small change in the format where earlier in --var-json (let's say for example VLAN_MEMBER), output format used to be-

Earlier-

 {
    "Vlan1000|Ethernet8": {
        "tagging_mode": "untagged"
    }
}

Just added the key as well to the output-

{
    "VLAN_MEMBER": {
        "Vlan1000|Ethernet8": {
            "tagging_mode": "untagged"
        }
    }
}